test/unit/connection/statement_test.js

```diff
@@ -2,425 +2,362 @@
  * Copyright (c) 2015 Snowflake Computing Inc. All rights reserved.
  */
 
-var Statement = require('./../../../lib/connection/statement');
-var ErrorCodes = require('./../../../lib/errors').codes;
-var assert = require('assert');
+var Statement = require("./../../../lib/connection/statement");
```

Comment:
do you change anything related to last query id in this file or is it just reformatting?

Reply:
No, nothing was changed. Just reformatting happened.

Reply:
let's reformat it fully (with lint:fix) or remove it from this PR and put in separate one since it's not related to the implemented feature

Reply:
I will remove it from this PR. I think it was added by mistake.

Reply:
As we need to replace the field name "statementId" with "queryId", this file has some changes. I applied eslint rules and added this in this PR.